pyshtools/shclasses.py

```diff
@@ -288,50 +291,53 @@ def from_random(self, power, kind='real', normalization='4pi', csphase=1):
                            csphase=csphase)
 
     @classmethod
-    def from_file(self, fname, lmax, format='shtools', kind='real',
+    def from_file(self, fname, lmax=None, format='shtools', kind='real',
                   normalization='4pi', csphase=1, **kwargs):
         """
-        Initialize the coefficients from input file.
+        Initialize the coefficients from a file.
 
         Usage
         -----
 
-        x = SHCoeffs.from_file(filename, lmax, [format, kind, normalization,
-                                                csphase, skip])
+        x = SHCoeffs.from_file(filename, lmax, [format='shtools', kind,
+                                                normalization, csphase, skip])
+        x = SHCoeffs.from_file(filename, [format='npy', kind, normalization,
+                                          csphase, skip])
```

Comment:
Based on the parameter description below, `skip` should not be necessary with `format='npy'`.